src/core/guards/__tests__/rate-limit.guard.spec.ts

```
@@ -0,0 +1,336 @@
+
+    it('should handle invalid environment variables gracefully', async () => {
+      process.env.RATE_LIMIT_AUTH_STRICT_MAX = 'invalid';
+      process.env.RATE_LIMIT_AUTH_WINDOW_MS = 'not-a-number';
+
+      expect(() => new AuthRateLimitGuard()).not.toThrow();
+
+      const guardWithInvalidEnv = new AuthRateLimitGuard();
+      const result = await guardWithInvalidEnv.canActivate(mockContext);
+      expect(result).toBe(true);
+
+      delete process.env.RATE_LIMIT_AUTH_STRICT_MAX;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)